### PR TITLE
Fix all the coverities detected last month

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This PR fixes all the coverities reported for the last 4 weeks, mainly addressing memory leaks

**Test plan**

Wait for travis/review the changes

**Closing issues**

none in here, but 15 in coverity scan

### CODE REVIEW

File: libr/main/rasign2.c [hunk 1]
```diff
@@ -139,11 +139,6 @@ R_API int r_main_rasign2(int argc, const char **argv) {
 		r_config_set_i (core->config, "scr.color", COLOR_MODE_DISABLED);
 	}
```
Comment (XVilka) [inline libr/main/rasign2.c:141]: Why did you remove that?

**Comment by trufae** [comment]
because it's dead code. few lines above that its doing the same, but using
the API instead of executing a command

**Comment by XVilka** [comment]
clang -c -MD   -fPIC -g -Wall -D__UNIX__=1 -I.. -DR2_PLUGIN_INCORE -Iarch -I/usr/home/build/radare2/libr/..//shlr -I/usr/home/build/radare2/libr/..//libr/asm/arch/include -I/usr/home/build/radare2/libr/../shlr/capstone/include -I../asm/arch/8051 -I../asm/arch/include -I../asm/arch/cr16/ -I../asm/arch/ebc/ -Iarch/gb/ -I../asm/arch/h8300/ -I../asm/arch/hexagon -I../anal/arch/hexagon -I../asm/arch/msp430/ -I../asm/arch/or1k -I -I../asm/arch/rsp -I../asm/arch/v810/ -I../asm/arch/v850/ -I -I/usr/home/build/radare2/libr -I/usr/home/build/radare2/libr/include -fvisibility=hidden -o bb.o bb.c
clang -c -MD   -fPIC -g -Wall -D__UNIX__=1 -I.. -DR2_PLUGIN_INCORE -Iarch -I/usr/home/build/radare2/libr/..//shlr -I/usr/home/build/radare2/libr/..//libr/asm/arch/include -I/usr/home/build/radare2/libr/../shlr/capstone/include -I../asm/arch/8051 -I../asm/arch/include -I../asm/arch/cr16/ -I../asm/arch/ebc/ -Iarch/gb/ -I../asm/arch/h8300/ -I../asm/arch/hexagon -I../anal/arch/hexagon -I../asm/arch/msp430/ -I../asm/arch/or1k -I -I../asm/arch/rsp -I../asm/arch/v810/ -I../asm/arch/v850/ -I -I/usr/home/build/radare2/libr -I/usr/home/build/radare2/libr/include -fvisibility=hidden -o var.o var.c
var.c:733:16: error: expected ';' at end of declaration
                char *varname anal->opt.varname_stack
                             ^
                             ;
1 error generated.
